nanopub/publication.py

```diff
@@ -45,8 +45,8 @@ def from_assertion(cls, assertion_rdf, uri=DEFAULT_NANOPUB_URI, introduces_conce
         derived from the nanopub's URI with a fragment (#) made from the blank node's name.
 
         Args:
-            derived_from: Add that this nanopub prov:wasDerivedFrom the given URI to the provenance
-                graph
+            derived_from: Add that this nanopub prov:wasDerivedFrom the given URI to the provenance graph.
+                          If a list of triples is passed, a provenance triple will be generated for each.
```

Comment:
A list of strings is passed right? Not triples?

Reply:
Ah sorry, yep, too tired... A list of strings or ```rdflib.URIRef``` are possible. Fixed now.